README.md

```diff
@@ -6,6 +6,9 @@
 [![.NetStandard 1.3](https://img.shields.io/badge/targets-.NETStandard%201.3-5C2D91?logo=.net)](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 [![Join the chat at https://gitter.im/sandialabs/Gulliver](https://badges.gitter.im/sandialabs/Gulliver.svg)](https://gitter.im/sandialabs/Gulliver?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://dev.azure.com/sandianationallabs/Gulliver/_apis/build/status/sandialabs.Gulliver?branchName=master)](https://dev.azure.com/sandianationallabs/Gulliver/_build/latest?definitionId=1&branchName=master)
+![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/sandianationallabs/Gulliver/1)
+![Azure DevOps tests (compact)](https://img.shields.io/azure-devops/tests/sandianationallabs/Gulliver/1?compact_message)
```

Comment:
How about a hard break in here keeping all the build stuff on its own line? It looks cluttered as is

![image](https://user-images.githubusercontent.com/1584973/67605824-bba8a180-f73c-11e9-8803-1886882d4fc8.png)

Reply:
![image](https://user-images.githubusercontent.com/440031/67606238-e515fd00-f73d-11e9-8b99-bcaa982026a3.png)